projects/project-player-library/src/lib/pages/main-player/main-player.component.css

```diff
@@ -1,7 +1,7 @@
 .inner-outlet {
     max-width: 700px;
-    min-height: 100vh;
-    padding: 10px ;
+    min-height: 97.5vh;
```

Comment:
No need to add this change, revert this file change

Reply:
done